### PR TITLE
Imi recipe

### PR DESCRIPTION
**Pull request by marksandford**
Added markup to PDF from @shauntru . Also expanded last section to walk through importing test objects.

### CODE REVIEW

File: docs/cookbook-recipes/multi_importer_installation.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+* The ingested objects should be listed.
+
+
+Other Useful links:
+[ISLE Documentation Site](https://github.com/Islandora-Collaboration-Group/ISLE)
+[More Sample Ingest Objects from Born Digital](https://github.com/Born-Digital-US/isle-ingest-samples)
+[IMI Github](https://github.com/mnylc/islandora_multi_importer)
+[Islandora Documentation - How to Add an Item to a Digital Collection](https://wiki.duraspace.org/display/ISLANDORA/How+to+Add+an+Item+to+a+Digital+Collection)
+[Islandora Documentation - How to Batch Ingest Files](https://wiki.duraspace.org/display/ISLANDORA7112/How+to+Batch+Ingest+Files)
```
Comment (dwk2) [inline docs/cookbook-recipes/multi_importer_installation.md]: need to add bullets here

File: docs/cookbook-recipes/multi_importer_installation.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
+# IMI Ingest: Setup and Use with ISLE
+
+These instruction will walk users through the installation of the Islandora Multi-Importer (IMI) Module and a test batch ingest of sample items.  They assume you have followed the instructions for [Local ISLE Installation: New Site](/install/install-local-new/).  Staging and Production servers may require a different entry in the docker-compose.yml volume section.
```
[the remark this answers is not in the document]
Comment (marksandford) [reply docs/cookbook-recipes/multi_importer_installation.md]: Done

File: docs/cookbook-recipes/multi_importer_installation.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+## Islandora ingest overview and IMI demonstration
+
+The script will have unpacked a set of sample files in the local ingest_data directory
+
+Drill down to the ./ingest_data/isle-ingest-samples/IMI directory and look for the following three files:
+
+mods_twig_base.txt
+testCol-tn.png.zip
+testCol.csv
```
[the remark this answers is not in the document]
Comment (dwk2) [reply docs/cookbook-recipes/multi_importer_installation.md]: one last tiny item here to resolve...

[the remark this answers is not in the document]
Comment (marksandford) [reply docs/cookbook-recipes/multi_importer_installation.md]: Sorry I missed that. Done

**Pull request merged**